src/perception/vision_pipeline/vision_pipeline/node_detector.py

```diff
@@ -51,19 +51,20 @@
 def cone_distance(
```

Comment:
ik its old code, but this function has a wack return...

`cone_distance` returns a number _and_ a rectangle?